FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -99,17 +99,27 @@ final class MessageViewController: ASViewController<TableNode> {
         let archiveInput = NavigationBarItemsView.Input(image: UIImage(named: "archive"), action: (self, #selector(handleArchiveTap)))
         let trashInput = NavigationBarItemsView.Input(image: UIImage(named: "trash"), action: (self, #selector(handleTrashTap)))
         let mailInput = NavigationBarItemsView.Input(image: UIImage(named: "mail"), action: (self, #selector(handleMailTap)))
-        let buttons: [NavigationBarItemsView.Input] = {
-            switch input?.path.lowercased() {
-            case MailDestination.Gmail.trash.path.lowercased():
-                return [infoInput, trashInput]
-            case MailDestination.Gmail.inbox.path.lowercased():
-                return [infoInput, archiveInput, trashInput, mailInput]
-            default:
-                return [infoInput, trashInput, mailInput]
+
+        navigationItem.rightBarButtonItem = NavigationBarItemsView(with: [infoInput])
+
+        imap.trashFolderPath()
```

Comment:
It would be good to add comments here - what each of the cases mean, and why is it done

Reply:
This is described in UI tests but will add it here as well to clarify